lib/usdUfe/ufe/Utils.cpp

```diff
@@ -278,27 +279,57 @@ int ufePathToInstanceIndex(const Ufe::Path& path, UsdPrim* prim)
     return instanceIndex;
 }
 
-std::string uniqueName(const TfToken::HashSet& existingNames, std::string srcName)
+bool splitNumericalSuffix(const std::string srcName, std::string& base, std::string& suffix)
```

Comment:
Split out the first part of the original unqiueName function which separated the input name into base and numerical suffix. Made into helper so I can call it elsewhere.